interface/therapy_groups/therapy_groups_controllers/therapy_groups_controller.php

```diff
@@ -0,0 +1,386 @@
+<?php
+/**
+ * interface/therapy_groups/therapy_groups_controllers/therapy_groups_controller.php contains the main controller for therapy groups.
+ *
+ * This is the main controller for the therapy group views and functionality.
+ *
+ * Copyright (C) 2016 Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * Copyright (C) 2016 Amiel Elboim <amielel@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * @author  Amiel Elboim <amielel@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+require_once dirname(__FILE__) . '/base_controller.php';
+require_once("{$GLOBALS['srcdir']}/appointments.inc.php");
+require_once("{$GLOBALS['srcdir']}/formatting.inc.php");
+require_once("{$GLOBALS['srcdir']}/pid.inc");
+
+class TherapyGroupsController extends BaseController{
+
+    public $therapyGroupModel;
+
+    //list of group statuses
+    public static $statuses = array(
+        '10' => 'active',
+        '20' => 'deleted',
```

Comment:
to translate these lists, which are used in status listing below, just do(since they are only used in views and don't go into the database, this is safe to do. also should capitalize them to make them look nicer on the list selector):
``````
        '10' => xl('Active'),
        '20' => xl('Deleted'),